earth_enterprise/src/fusion/gst/gstFormatManager.cpp

```diff
@@ -67,8 +67,9 @@ gstFormatManager::gstFormatManager() {
   RegisterFormat<gstKVPFormat>("Keyhole Vector Product",
                                "Keyhole Vector Product", "*.kvp ");
   RegisterFormat<gstGDALFormat>("Supported Raster", "GDAL",
-                                "*.tif *.TIF *.img *.IMG *.jp2 *.JP2 *.rpf"
-                                "*.RPF *.fit *.FIT *.dem *.DEM *.sid *.SID");
```

Comment:
Why was sid removed from here? We support it as a valid format if GDAL has the capability